src/Indexer.js

```diff
@@ -59,15 +61,50 @@ export default class Indexer {
     })
   }
 
+  /**
+   * Create indices, if needed, and add field mappings
+   * @returns {null}
+   */
+  async setupIndices() {
+    try {
+      for (const index of this.indices) {
+        const indexExists = await this.client.indices.exists({ index: index })
+
+        if (!indexExists) {
+          await this.client.indices.create({ index: index })
+        }
+
+        await this.client.indices.putMapping({
+          index: index,
+          type: Config.indexType,
+          body: {
+            properties: {
```

Comment:
Declare two field mappings. Keep titles and subtitles separate in the index (but we can search them at the same time, see above), and make them both full-text fields.

Reply:
@ndushay The field mappings? Hmmm, what do you have in mind? That we can add field mappings through config without touching the codebase?

Reply:
was pondering that, yes.  Probably too soon to do such a thing.

Reply:
Yeah, I do like that idea after mulling it over a bit more. Will make this change and bring in the `config` package (will close #19 as well).